account_banking_payment_export/wizard/payment_order_create.py

```diff
@@ -155,22 +155,20 @@ def _prepare_payment_line(self, payment, line):
             # -- end account banking
         # -- account_banking
         state = 'normal'
-        communication = line.ref or '-'
         if line.invoice:
-            if line.invoice.type in ('in_invoice', 'in_refund'):
-                if line.invoice.reference_type == 'structured':
-                    state = 'structured'
-                    communication = line.invoice.reference
-                else:
-                    if line.invoice.reference:
-                        communication = line.invoice.reference
-                    elif line.invoice.supplier_invoice_number:
-                        communication = line.invoice.supplier_invoice_number
-            else:
-                # Make sure that the communication includes the
```

Comment:
I have reorganized the code to compact it, because making 2 times the comparison with reference_type is not needed and the code is bigger, but I can do it the other way if you prefer.